[2023-05-24T16:38:18.508+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: refresh_token.ETL manual__2023-05-23T17:54:32.171118+00:00 [queued]>
[2023-05-24T16:38:18.522+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: refresh_token.ETL manual__2023-05-23T17:54:32.171118+00:00 [queued]>
[2023-05-24T16:38:18.522+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-24T16:38:18.522+0000] {taskinstance.py:1283} INFO - Starting attempt 28 of 32
[2023-05-24T16:38:18.523+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-24T16:38:18.543+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): ETL> on 2023-05-23 17:54:32.171118+00:00
[2023-05-24T16:38:18.551+0000] {standard_task_runner.py:55} INFO - Started process 2583 to run task
[2023-05-24T16:38:18.554+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'refresh_token', 'ETL', 'manual__2023-05-23T17:54:32.171118+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/refresh_token_DAG.py', '--cfg-path', '/tmp/tmp13kxq4s0']
[2023-05-24T16:38:18.556+0000] {standard_task_runner.py:83} INFO - Job 134: Subtask ETL
[2023-05-24T16:38:18.628+0000] {task_command.py:388} INFO - Running <TaskInstance: refresh_token.ETL manual__2023-05-23T17:54:32.171118+00:00 [running]> on host 2fa3ade57007
[2023-05-24T16:38:18.727+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=fox
AIRFLOW_CTX_DAG_ID=refresh_token
AIRFLOW_CTX_TASK_ID=ETL
AIRFLOW_CTX_EXECUTION_DATE=2023-05-23T17:54:32.171118+00:00
AIRFLOW_CTX_TRY_NUMBER=28
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-23T17:54:32.171118+00:00
[2023-05-24T16:38:18.743+0000] {logging_mixin.py:137} INFO - BQASrkZf0NnZeK23DpGXchbiLQkPA8rfq0Mqy3gnuj0iNgL7dP5bgU4gC2cGRtVwi9jPuH4wmXzMloSv51u7u6oZ8OWfWEoYhbi4JZZ6KMPfVy2KVvdY0Vsv8orsAxXLxTPe72xQ3ULH2ri9Vrz9zcecdAHRfdYCx1drALZiaCVMmLe5ByXeQ1R8u9lNIysByfgO_XjP05lM40M
[2023-05-24T16:38:19.267+0000] {python.py:177} INFO - Done. Returned value was:                   played_at  ...  timestamps
0  2023-05-23T14:34:26.395Z  ...  2023-05-23
1  2023-05-23T14:32:00.984Z  ...  2023-05-23
2  2023-05-23T14:29:08.305Z  ...  2023-05-23
3  2023-05-23T14:26:06.479Z  ...  2023-05-23
4  2023-05-23T14:23:49.778Z  ...  2023-05-23
5  2023-05-23T14:19:57.780Z  ...  2023-05-23
6  2023-05-23T14:17:55.047Z  ...  2023-05-23
7  2023-05-23T14:13:58.752Z  ...  2023-05-23
8  2023-05-23T09:36:44.159Z  ...  2023-05-23
9  2023-05-23T09:23:55.397Z  ...  2023-05-23

[10 rows x 4 columns]
[2023-05-24T16:38:19.301+0000] {xcom.py:635} ERROR - Object of type DataFrame is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-05-24T16:38:19.303+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2304, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 240, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2023-05-24T16:38:19.316+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=refresh_token, task_id=ETL, execution_date=20230523T175432, start_date=20230524T163818, end_date=20230524T163819
[2023-05-24T16:38:19.338+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 134 for task ETL (Object of type DataFrame is not JSON serializable; 2583)
[2023-05-24T16:38:19.367+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-24T16:38:19.399+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
